RawWaterManagement/CrossSection/examples/example-normalized.jsonld

```diff
@@ -0,0 +1,157 @@
+  },
+  "name": {
+    "type": "Property",
+    "value": "Blue garden message government recognize. One cause method often without ago."
+  },
+  "alternateName": {
+    "type": "Property",
+    "value": "Piece every miss culture positive. Yard concern current themselves house necessary appear."
+  },
+  "description": {
+    "type": "Property",
+    "value": "Inside big its increase stock chair. Daughter matter direction prepare player unit. Film behavior dream share employee water animal wall."
+  },
+  "dataProvider": {
+    "type": "Property",
+    "value": "Quickly blue moment she shoulder. But fact set."
+  },
+  "owner": {
+    "type": "Property",
+    "value": [
+      "urn:ngsi-ld:dataModel:items:IWWY:96787325",
+      "urn:ngsi-ld:dataModel:items:GIDC:83237831"
+    ]
+  },
+  "seeAlso": {
+    "type": "Property",
+    "value": [
+      "urn:ngsi-ld:dataModel:items:KABR:33050648",
+      "urn:ngsi-ld:dataModel:items:HQTS:22735104"
+    ]
+  },
+  "location": {
+    "type": "Property",
+    "value": {
+      "type": "Point",
+      "coordinates": [
+        -84.678881,
+        157.810024
+      ]
+    }
+  },
+  "address": {
+    "type": "Property",
+    "value": {
+      "streetAddress": "Evidence already goal seek party American nation. Floor leader pay answer south seven. Tonight through religious spring. Chance agent front participant true surface.",
+      "addressLocality": "Onto when I thousand environment start.",
+      "addressRegion": "Trip indicate over rest who. Significant environmental parent.",
+      "addressCountry": "Right class police any high. Loss certainly former after. Responsibility forward reason increase again measure.",
+      "postalCode": "Finally development animal sit often. Find believe realize option. Benefit manager join future require technology.",
+      "postOfficeBoxNumber": "Meeting push loss green increase happen meet month. Bar he anything source minute situation cover. Physical which beat current who likely white.",
+      "areaServed": "Professor anyone would nor. Cold road south thought fund line order."
+    }
+  },
+  "areaServed": {
+    "type": "Property",
+    "value": "Only list kid win. Run discussion shake scientist. Single read mission food."
+  },
+  "type": "CrossSection",
+  "attachedTo": {
+    "type": "object",
+    "value": "urn:ngsi-ld:dataModel:attachedTo:PUDY:66449777"
+  },
+  "observedBy": {
+    "type": "object",
+    "value": "urn:ngsi-ld:dataModel:observedBy:QKLD:19228496"
+  },
+  "tag": {
+    "type": "Property",
+    "value": "This write measure trial. Concern wife feeling five media less."
+  },
+  "position": {
+    "type": "Property",
+    "value": {
+      "distance": 864.6,
+      "refPoint": "urn:ngsi-ld:dataModel:refPoint:JXFD:60487647"
+    }
+  },
+  "waterFlow": {
+    "type": "Property",
+    "value": 864.6
+  },
+  "waterVelocity": {
+    "type": "Property",
+    "value": 864.6
+  },
+  "waterTemperature": {
```

Comment:
Use of unitcode is encouraged to ensure consistency of collected data. Otherwise, no one knows if e.g. a temperature is expressed in °C or K 
The specification recommends to take it from UN CEFACT (https://unece.org/DAM/cefact/recommendations/rec20/rec20.zip)